tools/pipelines/test-perf-benchmarks.yml

```
@@ -601,7 +601,7 @@ stages:
             FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
             FLUID_BUILD_ID: $(Build.BuildId)
             FLUID_ENDPOINTNAME: ${{ endpointObject.endpointName }}
-            FLUID_LOGGER_PROPS: '{ "hostName": "Benchmark" }'
+            FLUID_LOGGER_PROPS: '{ "hostName": "Benchmark", "displayName": ${{variables.pipelineIdentifierForTelemetry}} }'
```

I think we'll need those quotes too to ensure valid JSON for `JSON.parse()`. Can you update the rest?
```suggestion
            FLUID_LOGGER_PROPS: '{ "hostName": "Benchmark", "displayName": '${{variables.pipelineIdentifierForTelemetry}}" }'
```